packages/console/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@agoric/console",
+  "private": true,
+  "version": "0.1.0+1-dev",
+  "description": "Description forthcoming.",
+  "author": "Agoric",
+  "license": "Apache-2.0",
+  "type": "module",
+  "main": "./src/main.js",
+  "scripts": {
+    "lint": "eslint '**/*.js'",
+    "lint-fix": "eslint --fix '**/*.js'",
+    "test": "tap --no-esm --no-coverage --reporter spec test/**/*.test.js"
+  },
+  "dependencies": {
+    "@agoric/assert": "0.1.0"
+  },
+  "devDependencies": {
+    "tap": "14.10.5",
+    "tape": "5.0.1"
+  },
+  "files": [
+    "LICENSE*",
```

Comment:
No, please don't do that.  The source files are the source of truth right now.  Typescript's conversion to `.d.ts` is lossy, so we would not preserve the types as they are right now.

However, you can crib a `jsconfig.json` from `agoric-sdk/packages/notifier`, and look at the `package.json` to see the `lint` scripts that use `tsc` to verify the types against the sources.  Add `"*.js"` to `"files"` (so that `exported.js` is available if present) and it will Just Work for both Typescript and JS consumers.

Reply:
With @michaelfig 's "please don't do that" I'm resolving this conversation.